.coderabbit.yml

```
@@ -0,0 +1,104 @@
+    - v2 features target rewrite-v2 branch
+    - v1 fixes target main branch
+    - PRs should be under ~400 lines when feasible
+
+# Path-based configuration
+path_instructions:
+  - path: "app/api/v1/endpoints/agent/**"
+    instructions: |
+      This is Agent API v1 - IMMUTABLE CONTRACT.
+      Any changes must maintain exact compatibility with contracts/v1_api_swagger.json.
+      Validate against OpenAPI spec before approval.
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)